server/views/partials/breadCrumb.njk

```diff
@@ -1,11 +1,11 @@
 {{ govukBreadcrumbs({
```

Comment:
Minor - do we need this file as a partial at all now?
Are any pages actually bringing it in now?

Reply:
Yep makes sense to remove as it was only used on 1 page